packages/cli/src/index.ts

```diff
@@ -3,9 +3,9 @@ import { logger } from '@crackle/core/logger';
 import { resolveConfig } from '@crackle/core/resolve-config';
 import yargs from 'yargs';
 
-const setConfigOverrides = <T>(
+const setConfigOverrides = (
   config: CrackleConfig,
-  overrides: yargs.Arguments<T>,
+  overrides: CrackleConfig & Pick<yargs.Arguments, '_' | '$0'>,
```

Comment:
Not sure I understand this type. It's the crackle config plus the two random bits off the front of yargs?

Reply:
Yes, and see this comment https://github.com/seek-oss/crackle/pull/16#discussion_r949785835

I've now added `Flatten` utility type so you can see what's actually there.